tests/phpunit/bootstrap.php

```diff
@@ -3,3 +3,5 @@
 declare(strict_types=1);
 
 require __DIR__ . '/../../vendor/autoload.php';
+
+sleep(120);
```

Comment:
Jestli je to nutný, tak bych tam aspoň hodil comment proč.

Reply:
no je, protože ta dementní DB2 se přes ten wait tváří že je nastartovaná, ale ne zcela :) tak jsem to vyřešil tímhle